packages/modules/web-viewer-react/tsconfig.json

```diff
@@ -2,7 +2,9 @@
   "extends": "./node_modules/@itwin/build-tools/tsconfig-base.json",
   "compilerOptions": {
     "allowSyntheticDefaultImports": true,
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "moduleResolution": "node",
+    "module": "es2020"
```

Comment:
you should not need to override module here.

take a look at:
- https://github.com/iTwin/itwinjs-core/blob/master/tools/build/tsconfig-base.json
- https://github.com/iTwin/viewer/blob/next/packages/modules/viewer-react/package.json#L21

we typically extend the tsconfig-base (which should always have module `commonjs`)
then in the package.json scripts, in the build:cjs we build using the tsconfig in our repo
and when we build:esm, we override and set the module to `es2020`

i think the main thing you need to do is override the `target` to `es2020` in the tsconfig.json of every package